backend/tarsy/agents/prompts/builders.py

```
@@ -397,7 +397,7 @@
     def get_react_error_continuation(self, error_message: str) -> List[str]:
         """Get ReAct continuation prompts for error recovery."""
         return [
-            f"Observation: Error in reasoning: {error_message}. Please try a different approach.",
+            f"Observation: Error in reasoning: {error_message}. 🔥 REMEMBER: Use 'Thought:' (with colon). Please try a different approach.",
```

Ruff (0.12.2)

Line too long (140 > 88)

(E501)